redux-form/lib/actions.d.ts

```diff
@@ -138,7 +134,3 @@ export function unregisterField(form: string, name: string): Action;
  * Resets the 'touched' flag for all the fields passed in.
  */
 export function untouch(form: string, ...fields: string[]): Action;
-
-export function updateSyncErrors(from: string, syncErrors: FormErrors<FormData>, error: any): Action;
-
-export function updateSyncWarnings(form: string, syncWarnings: FormWarnings<FormData>, warning: any): Action;
```

Comment:
This is kind of confusing. Documentation says they are no longer present in the public API (http://redux-form.com/6.5.0/docs/api/ActionCreators.md/) but they are still present in the source for 6.5.0 (https://github.com/erikras/redux-form/blob/v6.5.0/src/actions.js). I wanted to be consistent with the official docs in case something gets removed or went to private API (which should be accessed directly).